packages/driver/cypress/e2e/commands/window.cy.js

```diff
@@ -145,7 +145,7 @@ describe('src/cy/commands/window', () => {
 
           expect(win).to.eq(this.win)
 
-          expect(this.logs[0].get('alias')).to.eq('win')
+          expect(this.logs[0].get('alias')).to.eq('@win')
```

Comment:
why did these change?

Reply:
I made the change so that we always display the '@' symbol in the command log, for both storing and retrieving - the consistency seemed more intuitive to me. It's not a necessary change and can be reverted without affecting the functionality if people disagree / discussion around it is going to delay the main features here.

It used to look like

```
// storing the alias
get     #button                            btn

// reading the alias
get     @btn
```

and now it's consistent

```
// storing the alias
get     #button                            @btn

// reading the alias
get     @btn
```

See the before/after screenshot in PR description (the 'after' shows the @ when both saving and reading the alias).